src/main/java/rs/hybridit/config/Configurations.java

```diff
@@ -0,0 +1,18 @@
+package rs.hybridit.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class Configurations {
```

Comment:
name is too generic, maybe SecurityConfiguration sounds better...?

Reply:
Renamed it.